Run.java

```diff
@@ -23,8 +23,18 @@
  * @author Jody Paul
  * @version 1.8.2
  */
-public class Run {
-    public static void main(String[] args) {
+public final class Run {
+    /**
+     * Default constructor - not used.
+     */
+    private Run() { }
+
+    /**
+     * Run the ImageLab application.
+     * @param args Command-line arguments; passed into the main
+     *             application
```

Comment:
This comment isn't particularly useful.  It should answer the question, "What are the command-line arguments to ImageLab?"